tokens/repository-map.json

```diff
@@ -152,7 +152,8 @@
         "tob-wgs",
         "sv-workflows",
         "production-pipelines",
-        "sample-metadata"
+        "sample-metadata",
+        "CellRegMap_pipeline"
```

Comment:
Can we rename this repository for naming consistency with the other repositories? E.g. `cell-reg-map-pipeline` or so, or even `CellRegMap-pipeline`, in case `CellRegMap` is an existing name.

Reply:
Thanks! I've renamed to cellregmap-pipeline :)